library/src/main/java/com/datatorrent/lib/db/jdbc/AbstractJdbcTransactionableOutputOperator.java

```diff
@@ -34,25 +33,30 @@
 import com.datatorrent.lib.db.AbstractPassThruTransactionableStoreOutputOperator;
 
 /**
- * This is the base class implementation of a transactionable JDBC output operator.&nbsp;
- * Subclasses should implement the method which provides the insertion command.
+ * This is the base class implementation of a transactionable JDBC output
```

Comment:
Please remove the formatting / check-style changes from this PR. Lets have just the relevant changes in this PR.